README.md

```
@@ -22,6 +37,8 @@ A dynamic Wake on Lan application that wakes servers based on the status of a NU
 
 ### 🛠️ Flexibility
 
+[something](#-getting-started)
```

markdownlint-cli2 (0.22.0)

[warning] Link fragments should be valid

(MD051, link-fragments)